### ISSUE TITLE
TmuxNavigator completly stopped working

### ISSUE THREAD

**Issue by 149-code** [christoomey/vim-tmux-navigator#353]
tldr; I've been using this plugin for a couple months and it had been working perfectly until it suddenly stopped working. None of the commands work (TmuxNavigateLeft, TmuxNavigateRight, ect...)

When I run TmuxNavigatorProcessList, nothing happens. My cursor just blinks and no process list appears. (See gif below)
When I run any of the Navigate commands nothing happens as well.

Tmux version: tmux 3.3a
Vim version: Vi IMproved 9.0 
OS: MacOS 13.4

![ezgif com-video-to-gif](https://github.com/christoomey/vim-tmux-navigator/assets/52003779/7c4525cd-f0e0-42f3-a474-cc7e2428259c)

If this isn't an obvious configuration error, I'd be happy to investigate the bug on my own (but I have no clue where to start so I would really appreciate advice on what might be going wrong).

Thanks in advance

**Comment by 149-code** [comment]
I did some more digging and I think my vim isn't able to run shell commands (one of my other plugins also broke) so I think this is probably a problem with my system and not with this plugin. Sorry for the false herring.
